asab/storage/__init__.py

```diff
@@ -26,5 +27,12 @@ def __init__(self, app):
 			from .elasticsearch import StorageService
 			self.Service = StorageService(app, "asab.StorageService")
 
+		# new configuration format
+		elif sttype is None:
```

Comment:
Nope, [asab:storage:elasticsearch] section must be specified if ElasticSearch should be used.
Even if it is empty.

Reply:
So if there is no 'type' in configurations you want it to default to elasticsearch?

Reply:
As discussed, it currently works for this fromat:

```
[asab:storage]
type=elasticsearch

[elasticsearch]
url=https://localhost:9200
```